pkg/process/io.go

```diff
@@ -255,11 +255,11 @@ func NewBinaryIO(ctx context.Context, id string, uri *url.URL) (_ runc.IO, err e
 		if err == nil {
 			return
 		}
-		result := multierror.Append(err)
+		result := []error{err}
 		for _, fn := range closers {
-			result = multierror.Append(result, fn())
+			result = append(result, fn())
 		}
-		err = multierror.Flatten(result)
+		err = errors.Join(result...)
```

Comment:
`multierror.Flatten` is only called here, so not sure if it's necessary. If so, maybe we can write a simple helper to do that.